echemdb/data/cv/entry.py

```diff
@@ -112,41 +112,161 @@ def __getitem__(self, name):
     def _descriptor(self):
         return Descriptor(self.package.descriptor)
 
-    def df(self, yunit=None):
+    def x(self):
+        r"""Return the x-axis name.
+
+        TODO: Adapt along with https://github.com/echemdb/svgdigitizer/issues/106.
+
+        EXAMPLES::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry.x()
+            'U'
+
+        """
+        from astropy import units as u
+        if u.Unit(self.figure_description.potential_scale.unit).is_equivalent('V'):
+            return 'U'
+
+    def y(self):
+        r"""Return the y-axis name.
+
+        TODO: Adapt along with https://github.com/echemdb/svgdigitizer/issues/106.
+
+        EXAMPLES::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry.y()
+            'j'
+
+        """
+        from astropy import units as u
+
+        if u.Unit(self.figure_description.current.unit).is_equivalent('A / m2'):
+            return 'j' 
+        if u.Unit(self.figure_description.current.unit).is_equivalent('A'):
+            return 'I'
+
+    def x_unit(self, xunit=None):
+        r"""Return an astropy unit, where `xunit` is a voltage.
+
+        EXAMPLES:
+
+        Without parameters, SI units are returned::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry.x_unit()
+            Unit("V")
+
+        When set to `"original"`, the original units of the published figure are returned::
+
+            >>> entry.x_unit(xunit='original')
+            Unit("V")
+
+        Units can be specified explicitly::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry.x_unit(xunit='mV')
+            Unit("mV")
+
+        """
+        from astropy import units as u
+
+        if xunit is None:
+            xunit = u.V
+        if xunit == 'original':
+            xunit = self.figure_description.potential_scale.unit
+        if isinstance(xunit, str):
+            xunit = u.Unit(xunit)
+
+        return xunit
```

Comment:
```suggestion

        return u.Unit(xunit)
```

xunit can already be a unit. isinstance is usually frowned upon in Python as it does not allow duck typing.